gradle.properties

```diff
@@ -13,7 +13,9 @@ org.gradle.jvmargs=-Xmx1G
 	archives_base_name = felt-api
 	felt-api-base-version=1.0.0
 	felt-ore-feature-api-v1-version=1.0.0
+	felt-splash-api-version=1.0.0
 	felt-shared-lifecycle-events-version=1.0.0
 
 # Dependencies
 	fabric_version=0.47.8+1.18.2
+	modmenu_version=3.1.0
```

Comment:
You sure we should dep modmenu?

Prospector is kind of not on the side of Fabric...

Reply:
we need it for testing, if we have something better id dep it here, but its just so much either to use than reading logs to check whats loaded

Reply:
I did hear they were going to just going to merge it into qsl, so honestly it'll probably just die on fabric anyways. Imo modmenu should be part of fabric api

Reply:
MM shouldn't be apart of FAPI imo thats out of scope

Reply:
The real Fabric API was the friends and memories we made along the way. <3

Reply:
indeed

Reply:
forge has a lot of things as one big package yeah, fabric doesn't really need that, though id be open to a mod menu api, that doesnt include the menu just data mods need to give whatever mod menu in felt, so implementing different menus is either, but that's offtopic